test-lit/test_splitter.py

```diff
@@ -0,0 +1,126 @@
+""" test_splitter.py
+
+    Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+    See https://llvm.org/LICENSE.txt for license information.
+    SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+    
+"""
+
```

Comment:
I would like if these 2 new python files had a little explanation of Why they exists and what they do.
The header I believe the way it is, is not needed

Reply:
I agree with the explanation, I'm not 100% sure about the license, since I was told to add it, but I'll verify it's needed, since the tests no longer have any license in them.